recipes/cccl/all/conanfile.py

```diff
@@ -18,50 +22,55 @@ class CcclConan(ConanFile):
         "muffle": True,
         "verbose": False,
     }
-    settings = "compiler",
 
-    _source_subfolder = "source_subfolder"
+    @property
+    def _cccl_dir(self):
+        return os.path.join(self.package_folder, "bin")
 
     def configure(self):
-        if self.settings.compiler != "Visual Studio":
-            raise ConanInvalidConfiguration("This recipe support only Visual Studio")
+        if not is_msvc(self):
+            raise ConanInvalidConfiguration("This recipe only supports msvc/Visual Studio.")
         del self.settings.compiler
 
+    def layout(self):
+        basic_layout(self, src_folder="source_subfolder")
+
     def package_id(self):
         del self.info.options.muffle
         del self.info.options.verbose
```

Comment:
```suggestion
        del self.info.options.verbose
        self.info.settings.clear()
```
Drop all settings, the package always contain the same file

Reply:
It can even be replaced by `self.info.clear()`

Reply:
Since this is a tool i'd expect it to need os and arch 🤔 right now its one package for windows linux and mac which seems a touch odd

Reply:
The idea with this tool is that it "translates" cc calls (gcc, g++) to cl calls (msvc). As such, it only makes sense on windows and when msvc is the requested compiler. What's the best way to limit it to that use?